examples/time_series_classification.ipynb

```diff
@@ -18,6 +18,7 @@
     "\n",
     "## See also\n",
     "\n",
+    "- [Gravitational waves detection](https://giotto-ai.github.io/gtda-docs/latest/notebooks/gravitational_waves_detection.html), where the *Takens embedding* technique helps us find recurrent structures in noisy signals with gravitational waves, and detect their presence.\n",
```

Comment:
+1 for this! Just a small thing: maybe "noisy signals from gravitational waves"

Reply:
> I'm thinking that it makes little sense for the Topology in time series notebook to now be called `time_series_classification` as there is nothing on classification in there any more. What do you think @wreise @lewtun? We could rename it to `topology_time_series` for instance.

You are right - I followed your suggestion